certbot/main.py

```diff
@@ -405,6 +405,17 @@ def _init_le_client(config, authenticator, installer):
 
     return client.Client(config, acc, authenticator, installer, acme=acme)
 
+def _want_delete(config):
+    """Does the user want to delete their now-revoked certs?
+
+    """
+    prompt = (
+            'Would you like certbot to delete the revoked certificate '
+            'and related files on disk? Warning: your severs could be '
```

Comment:
I was thinking we should leave it out because it might cause people to worry about something where they don't know what's going on, where they've probably already got it. If anything, maybe, "you will also need to update your server's configuration files to stop serving a nonexistent file." But not sure we need anything here at all.